test/unit/test_cli.py

```diff
@@ -121,15 +121,15 @@ def test_cli_10(self):
         result = self.runner.invoke(app, [])
         self.assertEqual(result.exit_code, 0, (result, result.stdout))
         self.mock_os_cloud.compute.create_server_group.assert_called_with(
-            name="stress_test_server_group",
+            name="simple-stress",
             policies=["soft-anti-affinity"],
         )
 
     def test_cli_11(self):
         result = self.runner.invoke(app, ["--affinity=anti-affinity"])
         self.assertEqual(result.exit_code, 0, (result, result.stdout))
         self.mock_os_cloud.compute.create_server_group.assert_called_with(
-            name="stress_test_server_group",
+            name="simple-stress",
             policies=["anti-affinity"],
         )
 
```

Comment:
@gndrmnn Do we have to add self.mock_os_cloud.compute.delete_server_group.assert_called_with(...) here as well?

Reply:
Yes, both affinity test cases should check if the ``delete_server_group`` was called